### PR TITLE
Adjust to Deflect 0.12 v5 API, handle new PAN event

### PR DESCRIPTION
**Pull request by rdumusc**
Needs: https://github.com/BlueBrain/Deflect/pull/130

### CODE REVIEW

File: eq/deflect/eventHandler.cpp [hunk 1]
```diff
@@ -227,18 +233,20 @@ void EventHandler::_processEvents( const Proxy* proxy )
...
+            event.type = Event::CHANNEL_POINTER_WHEEL;
+            event.pointerWheel.x = x;
+            event.pointerWheel.y = pvp.h - y;
+            event.pointerWheel.buttons = PTR_BUTTON_NONE;
+            const auto dx = deflectEvent.dx * pvp.w;
+            const auto dy = deflectEvent.dy * pvp.h;
+            const auto sign = dx + dy;
+            const auto zoom = std::copysign( std::sqrt( dx*dx + dy*dy ), sign );
+            event.pointerWheel.xAxis = 0.f;
+            event.pointerWheel.yAxis = zoom / 40.f;
```
Comment (tribal-tec) [inline eq/deflect/eventHandler.cpp]: where does the magic 40 come from?

File: eq/deflect/eventHandler.cpp [hunk 2]
```diff
@@ -202,6 +202,12 @@ void EventHandler::_processEvents( const Proxy* proxy )
         case ::deflect::Event::EVT_DOUBLECLICK:
             break;
         case ::deflect::Event::EVT_MOVE:
+             // no break;
+        case ::deflect::Event::EVT_PAN:
+            _proxy->setNavigationMode(
```
Comment (tribal-tec) [inline eq/deflect/eventHandler.cpp]: do we still need that mode + the print?

File: eq/deflect/eventHandler.cpp (hunk 1 above)
Comment (rdumusc) [inline eq/deflect/eventHandler.cpp]: It's a bit of magic that was copied from the wheel event above and I cannot say that I understand what it corresponds to. Documentation for xAxis states: 'wheel rotation in clicks'. That value is then multiplied again by 0.05f in eqPly / seq / livre to move the camera...

File: eq/deflect/eventHandler.cpp (hunk 2 above)
Comment (rdumusc) [inline eq/deflect/eventHandler.cpp]: If the mode is not set, the camera keeps rotating. I'm having a look if the print can be disabled

**Comment by rdumusc** [comment]
Turns out the print and the rotation mode were specific to Deflect, so I removed them. Now the Channel::drawOverlay() looks suspiciously empty, but I didn't remove the applyOverlayState() and  resetOverlayState() calls just to be safe. Let me know if you think they can be removed too.

**Comment by tribal-tec** [comment]
They can be removed too then. Just NOP it like frameViewStart() a few lines above.

**Comment by rdumusc** [comment]
OK, done!

**Comment by tribal-tec** [comment]
Typo 'Delfect' in PR title and commit msg

**Comment by tribal-tec** [comment]
+1 otherwise

**Pull request merged**